### PR TITLE
Make 'git-elegant' calls in scripts unified

### LINKED ISSUE #74: Use 'git-elegant' instead of 'git elegant'
In scripts `git-elegant-clone` and `git-elegant-init` there are commands:
```
git elegant configure --local
```
It's working fine with such calls, but to keep unified structure we have to make them:
```
git-elegant configure --local
```

### PR DESCRIPTION
**Pull request by AntonKarabaza**
Use 'git-elegant' calls instead of 'git elegant'.

Closes #74

### CODE REVIEW

File: src/test/git-elegant-clone.bats [hunk 1]
```diff
@@ -7,7 +7,13 @@ load fake-cd
 setup() {
     fake-pass git clone
     fake-pass git "clone https://github.com/extsoft/elegant-git.git"
-    fake-pass git "elegant configure --local"
```
Comment (extsoft) [inline src/test/git-elegant-clone.bats]: It's a bad idea to replace `git elegant configure --local` mock with the mocks for everything this command does. Please find another way.

File: src/test/git-elegant-init.bats [hunk 2]
```diff
@@ -5,7 +5,13 @@ load fake-read
 
 setup() {
     fake-pass git init
-    fake-pass git "elegant configure --local"
```
Comment (extsoft) [inline src/test/git-elegant-init.bats]: Please see comment for `src/test/git-elegant-clone.bats`

**Comment by AntonKarabaza** [comment]
@extsoft Seems that this pull request is no longer necessary.

**Comment by extsoft** [comment]
@AntonKarabaza #74 isn't solved. Is the task incorrect?

**Comment by AntonKarabaza** [comment]
@extsoft Yes, we had two opposite by their aims pull requests: the current one and this #87. The #87 is already approved and merged, so the current one is redundant.

**Pull request closed without merging**